### PR TITLE
[feature] : Add contact us

### PR DESCRIPTION
**Pull request by Dhananjayhub**
## Description

-   [Ticket Link](#) (if any)
-   Provide a small description ( 2 - 3 lines) of the Pull Request
    Add contact us
## UI screenshot or Video
[screen-capture (4).webm](https://github.com/primathonacademy/april-batch-live-project/assets/71414293/307259d5-7bb2-433e-b56d-0bdd5fa8679b)

-   Please include a screenshot or video to help reviewers understand the specific part of the application being modified. Multiple screenshots are appreciated

---

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,13 @@
     },
     "dependencies": {
         "@gsap/react": "^2.1.0",
+        "formik": "^2.4.5",
```
Comment (ram-primathon) [inline package.json:16]: don't use formik

File: package.json [hunk 2]
```diff
@@ -13,11 +13,13 @@
...
         "prop-types": "^15.8.1",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "split-type": "^0.3.4",
-        "tailwind-scrollbar-hide": "^1.1.7"
+        "tailwind-scrollbar-hide": "^1.1.7",
+        "yup": "^1.4.0"
```
Comment (ram-primathon) [inline package.json:22]: don't use yup

**Review by ram-primathon — CHANGES_REQUESTED** [review]

**Comment by ram-primathon** [comment]
Please implement native using react form

**Comment by ram-primathon** [comment]
As you can see we are using just one form so there is no point in adding an external package for this.

File: package.json (hunk 1 above)
Comment (Dhananjayhub) [reply package.json:16]: ok sir

File: package.json (hunk 2 above)
Comment (Dhananjayhub) [reply package.json:22]: ok sir

**Pull request closed without merging**